content/manager/security.md

```diff
@@ -721,19 +750,53 @@ my_manager_blueprint:
 {{% gsNote title="Handling system dependencies" %}}
 The LDAP userstore driver example uses [python-ldap](http://www.python-ldap.org/doc/html/ldap.html#module-ldap).
 
-Ideally, we would like to run this command on plugin installation:
+Ideally, we would like to run this command before plugin installation:
 {{< gsHighlight  bash >}}
-sudo yum -y install python-ldap
+sudo yum install python-devel openldap-devel gcc -y
 {{< /gsHighlight >}}
 
 Unfortunately, currently there is no convenient way to specify system dependencies as plugin requirements. This is a
 known issue and is intended to be solved in future versions.
 
 To work around it, edit the REST creation script of the selected manager blueprint
-(*"/components/restservice/scripts/create.sh"* relative to the main manager blueprint file directory). In this script, add the command required to install
+(*"/components/restservice/scripts/create.sh"* relative to the main manager blueprint file directory). In this script, add the above command required to install
 [python-ldap](http://www.python-ldap.org/doc/html/ldap.html#module-ldap) just before the REST service installation
 command.
 
 Alternatively, modify the relevant manager blueprint to include the installation of the required system dependencies.
 
 {{% /gsNote %}}
+
+#### LDAP authentication provider example:
+The [cloudify-ldap-plugin](https://github.com/cloudify-cosmo/cloudify-ldap-plugin) provides the ability to authenticate users against any LDAP endpoint.
+Configuring the ldap authentication driver:
+{{< gsHighlight  yaml  >}}
+authentication_providers:
+  implementation: authentication.ldap_authentication_provider:LDAPAuthenticationProvider
+  name: ldap_authentication_provider
+  properties:
+    'directory_url': ldap://x.x.x.x:389
+{{< /gsHighlight >}}
+**Installation**<br>
+Since the [cloudify-ldap-plugin](https://github.com/cloudify-cosmo/cloudify-ldap-plugin) is not installed by default when preforming bootstrap, a [custom rest plugin](#packaging-configuring-and-installing-custom-implementations)
+must be defined in the manager-blueprint, that would be uploaded and installed upon bootstrap.
+Defining the [cloudify-ldap-plugin](https://github.com/cloudify-cosmo/cloudify-ldap-plugin) in the manager blueprint as a rest plugin:
+{{< gsHighlight  yaml  >}}
+node_types:
+  ...
+  manager.nodes.RestService:
+    ...
+    properties:
+      ...
+      plugins:
+        ldap_authentication_provider:
+          source: https://github.com/cloudify-cosmo/cloudify-ldap-plugin/archive/1.0.zip
+          install_args: '--pre'
+{{< /gsHighlight >}}
+**System-level dependencies**<br>
+The LDAP python dependency `python-ldap`, included in the [cloudify-ldap-plugin](https://github.com/cloudify-cosmo/cloudify-ldap-plugin) project, requires system level dependencies
+e.g openldap-devel, python-devel, and gcc in order to install.
```

Comment:
since these are required dependencies, 'e.g.' may be misleading.
I'd be more explicit by saying, 'requires the following system level dependencies: openldap-devel,...'